##[error]Error: Cannot find module 'handlebars'

**Describe the bug**
In Custom Handlebars Extension, I try to add a function to sort array by 2 fields like in example AzurePipelines/SampleTemplates/XplatGenerateReleaseNotes (Node based)/Version 3/each_with_sort_by_field.js 


        const handlebars = require("handlebars");
        module.exports = {get_system_type_sort_by_field(relatedWorkItems, typeName1, typeName2, key1, key2, opts) {
            if(opts.data) {
                data  = handlebars.createFrame(opts.data) 
            }

            var foundList = [];
            for(var arrayCount = 0; arrayCount < relatedWorkItems.length ; arrayCount++){
                if(relatedWorkItems[arrayCount].fields['System.WorkItemType'] === typeName1 || 
                relatedWorkItems[arrayCount].fields['System.WorkItemType'] === typeName2){
                    var system_type = relatedWorkItems[arrayCount];
                    if (!foundList.includes(system_type)) {
                        foundList.push(system_type);
                    }
                }
            }

            // sort by 2 field
            foundList = foundList.sort (function (a, b) {
                var a1 = a.fields[key1]
                var b1 = b.fields[key1]
                var a2 = a.fields[key2]
                var b2 = b.fields[key2]

                if(a1 == b1){
                    return (a2 < b2) ? -1 : (a2 > b2) ? 1 : 0;
                }else{
                    return (a1 < b1) ? -1 : 1;
                }
            });
    
        
            // the iterator
            var len = foundList.length;
            var buffer = '';
            var i = -1;
    
            while (++i < len) {
            var item = foundList[i];
            data.index = I;
            item.index = i + 1;
            item.total = len;
            item.isFirst = i === 0;
            item.isLast = i === (len - 1);
            buffer += opts.fn(item, {data: data});
            }
            return buffer;
            }
        };


The first line const handlebars = require("handlebars")
error message ##[error]Error: Cannot find module 'handlebars' 

In the template, I use the function 
        {{#get_system_type_sort_by_field this.relatedWorkItems "Bug" "Product Backlog Item" "System.Title" "System.Id"}}
        - {{this.id}}  - {{lookup this.fields 'System.WorkItemType'}} - {{lookup this.fields 'System.State'}} 
        {{/get_system_type_sort_by_field}}

--- a/Extensions/XplatGenerateReleaseNotes/V3/ReleaseNotesFunctions.ts
+++ b/Extensions/XplatGenerateReleaseNotes/V3/ReleaseNotesFunctions.ts
@@ -589,7 +589,9 @@ export function processTemplate(
             if (!customHandlebarsExtensionFolder || customHandlebarsExtensionFolder.length === 0) {
                 // cannot use process.env.Agent_TempDirectory as only set on Windows agent, so build it up from the agent base
                 // Note that the name is case sensitive on Mac and Linux
-                customHandlebarsExtensionFolder = `${process.env.AGENT_WORKFOLDER}/_temp`;
+                // Also #832 found that the temp file has to be under the same folder structure as the main .js files
+                // else you cannot load any modules
+                customHandlebarsExtensionFolder = __dirname;
             }
 
             agentApi.logInfo("Loading custom handlebars extension");

